packages/terra-table/docs/guides/MultiSelectTable.md

```diff
@@ -0,0 +1,127 @@
+# Terra Table - Implementing a Multi Select Table
+
+In previous versions of the terra-table a multi select style table could be created as either a controlled version of SelectableTableRows or uncontrolled version of MultiSelectableRows.  These implementations suffered from inflexibility and performance concerns. Going foward terra-table is more granular, though this puts more reponsibility on the consumer to properly update their table rows with the appropriate state. The following is a guide to addressing those concerns in your implementation.
+
+## State Management
+The state of selection needs to be managed for the table in a HOC. In this example we are going to be a unique key, but the type of state used is open to the implementor of the HOC.
```

Comment:
For the first instance where it's used, it maybe be better to specify High Order Component (HOC)?

Reply:
Updated.